scripts/compose.py

```diff
@@ -1964,7 +2009,7 @@ def _content(self):
                 "ELASTIC_APM_SOURCE_LINES_ERROR_LIBRARY_FRAMES",
                 "ELASTIC_APM_SOURCE_LINES_SPAN_LIBRARY_FRAMES",
                 "REDIS_URL=redis://redis:6379",
-                "ELASTICSEARCH_URL=http://elasticsearch:9200",
+                "ELASTICSEARCH_URL={}".format(self.es_urls),
```

Comment:
like the logstash case, needs to be joined or an entry picked

Reply:
https://github.com/elastic/apm-integration-testing/pull/557#discussion_r306676939